ts/screens/wallet/payment/PickPaymentMethodScreen.tsx

```diff
@@ -116,6 +124,16 @@ class PickPaymentMethodScreen extends React.Component<Props> {
 
         <View spacer={true} />
 
+        {wallets.some(myWallet => myWallet.creditCard?.brand === "AMEX") &&
+          verifica.importoSingoloVersamento >= 100000 && (
```

Comment:
Could 10000 be a const declared outside the component?
In that way it should be easier to read/edit
https://en.wikipedia.org/wiki/Magic_number_(programming)